.misc/deploy.testpypi.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Imo you should improve deploy.pypi.sh , maybe using two env vars to control

1. Whether test or production is used,
2. Whether rultor secrets are needed